sdk_contrib/typed_span/src/main/java/io/opentelemetry/sdk/contrib/typedspan/http/HttpSpanWrapperBuilder.java

```diff
@@ -0,0 +1,111 @@
+package io.opentelemetry.sdk.contrib.typedspan.http;
+
+import io.opentelemetry.sdk.contrib.typedspan.BaseSpanWrapper;
+import io.opentelemetry.sdk.contrib.typedspan.BaseSpanWrapperBuilder;
+import io.opentelemetry.trace.Tracer;
+
+public abstract class HttpSpanWrapperBuilder extends BaseSpanWrapperBuilder<BaseSpanWrapper> {
+
+  static final String UNKNOWN_URL = "<unknown>";
+
+  /**
+   * Semantic Attributes
+   * https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-http.md
+   */
+  public static final String METHOD_KEY = "http.method";
```

Comment:
This should later be switched to use #758.

Reply:
This is just a prototype to showcase how the API should look like. It is not meant to be merged